rpc/api.proto

```diff
@@ -743,3 +743,53 @@ message VerifyMessageRequest {
 message VerifyMessageResponse {
 	bool valid = 1;
 }
+
+service DecodeMessageService {
+	rpc DecodeRawTransaction (DecodeRawTransactionRequest) returns (DecodeRawTransactionResponse);
+}
+
+message DecodedTransaction {
+	message ScriptSig {
+		string asm = 1;
+		string hex = 2;
+	}
+	message ScriptPubKeyResult {
+		string asm = 1;
+		string hex = 2;
```

Comment:
`bytes`, and it makes the most logical sense for this to be the first field in the message.